web/ui/react-app/src/pages/graph/Graph.tsx

```diff
@@ -57,6 +58,10 @@ class Graph extends PureComponent<GraphProps, GraphState> {
         }
       });
     }
+
+    if (prevProps.useLocalTime !== useLocalTime) {
+      this.plot();
```

Comment:
Shouldn't the displayed data (if have such) to be restored? ```this.plot(this.$chart.getData() as GraphSeries[]);```

Reply:
I'm not sure what you mean. `plot()` currently defaults to `this.state.chartData`, which is already stored. Why would I need to get it out of `this.$chart.getData()`?

Reply:
Ah, yes. Just didn't check how ```plot```is implemented, instead I saw how data is passed in ```handleResize``` and I thought it should be done the same way, anyways, ignore it :)

Reply:
Everything else looks good to me :+1: